.travis.yml

```diff
@@ -4,8 +4,7 @@ node_js:
   - '12'
 before_install:
   - npm install -g grunt-cli
-before_script:
-  - grunt build
 script: 
+  - npm run build
```

Comment:
i'm going ot try this as a before_script so we are sure it's done before it tries the tests.